en/advanced_config/parameter_reference.md

```diff
@@ -14037,6 +14037,22 @@ How often the sensor is readout</p>   <p><b>Reboot required:</b> true</p>
  <td style="vertical-align: top;">1</td>
  <td style="vertical-align: top;"></td>
 </tr>
+<tr>
```

Comment:
Please remove these changes. The parameters are updated automatically from firmware source code.